includes/class-micropub-endpoint.php

```diff
@@ -349,6 +339,23 @@ public static function query_handler( $request ) {
 		return new WP_REST_Response( $resp, 200 );
 	}
 
+	public static function query( $post_id ) {
```

Comment:
mind adding a comment describing what this does, args and return value/type, etc?